addons/web/static/src/core/dialog/dialog.scss

```diff
@@ -1,52 +1,11 @@
-.o_dialog {
-  > .modal {
-    display: block;
+.modal.o_technical_modal .modal-content .modal-footer footer {
+  > :not(:first-child) {
+    margin-left: map-get($spacers, 1);
```

Comment:
I really don't like this kind of magic :/
Could you review all templates injecting data in `footerTemplate` and apply margin classes directly?
eg https://github.com/odoo/odoo/blob/master/addons/web/static/src/core/debug/debug_menu_items.xml#L4